modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/macros/EVPOrganization.macro

```diff
@@ -0,0 +1,156 @@
+		}
+
+		if (isSet(smallDescription)) {
+			Type(
+				key_fieldLabel = "Small Description",
+				locator1 = "TextInput#GENERIC_TEXT_INPUT",
+				value1 = ${smallDescription});
+		}
+
+		if (isSet(street)) {
+			Type(
+				key_fieldLabel = "Street",
+				locator1 = "TextInput#GENERIC_TEXT_INPUT",
+				value1 = ${street});
+		}
+
+		if (isSet(city)) {
+			Type(
+				key_fieldLabel = "City",
+				locator1 = "TextInput#GENERIC_TEXT_INPUT",
+				value1 = ${city});
+		}
+
+		if (isSet(state)) {
+			Type(
+				key_fieldLabel = "State",
+				locator1 = "TextInput#GENERIC_TEXT_INPUT",
+				value1 = ${state});
+		}
+
+		if (isSet(country)) {
+			Type(
+				key_fieldLabel = "Country",
+				locator1 = "TextInput#GENERIC_TEXT_INPUT",
+				value1 = ${country});
+		}
+
+		if (isSet(zip)) {
+			Type(
+				key_fieldLabel = "Zip",
+				locator1 = "TextInput#GENERIC_TEXT_INPUT",
+				value1 = ${zip});
+		}
+
+		if (isSet(taxId)) {
+			Type(
+				key_fieldLabel = "Tax Id",
+				locator1 = "TextInput#GENERIC_TEXT_INPUT",
+				value1 = ${taxId});
+		}
+
+		if (isSet(contactName)) {
+			Type(
+				key_fieldLabel = "Contact Name",
+				locator1 = "TextInput#GENERIC_TEXT_INPUT",
+				value1 = ${contactName});
+		}
+
+		if (isSet(contactEmail)) {
+			Type(
+				key_fieldLabel = "Contact Email",
+				locator1 = "TextInput#GENERIC_TEXT_INPUT",
+				value1 = ${contactEmail});
+		}
+
+		if (isSet(contactPhone)) {
+			Type(
+				key_fieldLabel = "Contact Phone",
+				locator1 = "TextInput#GENERIC_TEXT_INPUT",
+				value1 = ${contactPhone});
+		}
+
+		if (isSet(bankingInfo)) {
+			Type(
+				key_fieldLabel = "Banking Info",
+				locator1 = "TextInput#GENERIC_TEXT_INPUT",
+				value1 = ${bankingInfo});
+		}
+
+		ScrollWebElementIntoView(locator1 = "Button#SUBMIT");
+
+		Button.click(button = "Submit");
+
+		if (isSet(createRequest)) {
+			Button.click(button = "Create a Request");
+		}
+		else {
+			Button.click(button = "Done");
+		}
+	}
+
+	@summary = "Add a new organization"
+	macro goToCreateOrganization {
+		EVPNavigator.goToNavItem(navItem = "Organization");
+
+		Button.click(button = "Create an Organization");
+	}
+
+	@summary = "Click kebab menu in organization page"
+	macro kebabMenuOrganization {
+		Click(
+			key_name = ${organizationName},
+			locator1 = "EVPUtils#KEBAB_MENU_OPTION_ORGANIZATION");
+
+		MenuItem.click(menuItem = ${kebabOption});
+	}
+
+	@summary = "Review request to organization page"
+	macro reviewOrganization {
+		Type(
+			key_fieldId = "messageDescribed",
+			locator1 = "TextArea#ANY_ID",
+			value1 = ${typeMessage});
+
+		Button.click(button = ${button});
+	}
+
+	@summary = "Action kebab menu is not present"
+	macro viewOptionNotPresentInKebabMenu {
+		if (IsElementPresent(key_name = ${organizationName}, locator1 = "EVPUtils#KEBAB_MENU_OPTION_ORGANIZATION")) {
+			Click(
+				key_name = ${organizationName},
+				locator1 = "EVPUtils#KEBAB_MENU_OPTION_ORGANIZATION");
+
+			MenuItem.viewNotPresent(menuItem = ${menuItem});
+		}
+	}
```

Comment:
Hi @joelrocha68, as your macro just do one thing, this `if` condition isn't necessary. Could you please remove it?